src/pages/VenuePage/VenuePage.tsx

```diff
@@ -224,21 +351,41 @@ export const VenuePage: React.FC = () => {
 
     if (isEventStartingSoon(event)) {
       return (
-        <CountDown
-          startUtcSeconds={event.start_utc_seconds}
-          textBeforeCountdown="Bar opens in"
-        />
+        <>
+          <Preload venue={venue} />
+          <Suspense fallback={<></>}>
+            <CountDown
+              startUtcSeconds={event.start_utc_seconds}
+              textBeforeCountdown="Bar opens in"
+            />
+          </Suspense>
+        </>
       );
     }
   }
 
-  if (!user || isRelatedVenuesLoading) {
-    return <LoadingPage />;
+  // @debt there is already !user check above, this is superfluous
+  if (!user) {
+    return (
+      <>
+        <Preload venue={venue} />
+        <Suspense fallback={<></>}>
+          <LoadingPage />
+        </Suspense>
+      </>
+    );
   }
```

Comment:
If it's not needed, then remove it?

Reply:
At this stage i'm not convinced that `Preload` is going to have any meaningful impact on things; and if we do decide to keep it, it shouldn't end up duplicated in so many places.

Reply:
Lazy loading `LoadingPage` and wrapping it with `Suspense` here doesn't make any sense.

As per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
Removed the second `!user`  in https://github.com/sparkletown/sparkle/pull/1713/commits/b7ec26dc13ee3c39f4d061fe3565ff7cd02b0f3e.

First one is enough.